src/form-saving-extensions/form-saving-extensions-form.js

```diff
@@ -163,7 +163,22 @@ angular.module( 'ngynFormSavingExtensions' ).directive( 'form', function() {
               }
               else {
                 var isPropertyError = false;
-                ( error.propertyNames || [] ).forEach( function( propertyName ) {
+
+                var propertyNames = [];
+                var errorPropertyNames = error.propertyNames || [];
+                var errorEntityIndexes = error.entityIndexes || [];
```

Comment:
This seems ambiguous as to how the property name matches up with the index.

Could entityIndex be singular?

If this remains plural, the correct pluralization in this context is typically considered to be `indices`